File: Ruby/easy/easy9/welcome_stranger.rb
# Exericse 1

def greetings(array, hash)
  puts "Hello #{array.join(' ')}! It's nice to have a #{hash[:title]} #{hash[:occupation]} around!"
end

greetings(['John', 'Q', 'Doe'], { title: 'Master', occupation: 'Plumber' })
#=> Hello, John Q Doe! Nice to have a Master Plumber around.

File: javascript/js-101-small-problems/easy2/welcome-stranger.js
function greetings(arr, obj) {
  return `Hello, ${arr.join(' ')}! Nice to have a ${obj.title} ${obj.occupation} around.`;
}


console.log(
  greetings(["John", "Q", "Doe"], { title: "Master", occupation: "Plumber" })
);
// logs Hello, John Q Doe! Nice to have a Master Plumber around.
